_rules/word-spacing-not-important-9e45ec.md

```diff
@@ -0,0 +1,203 @@
+---
```

Comment:
Big problem with this rule. See https://github.com/act-rules/act-rules.github.io/pull/1412#issuecomment-733606665